### PR TITLE
Added desert 404 template

### LINKED ISSUE #18: New 404 Templates
Any new 404 template is welcome.

**Only templates using [tsParticles](https://particles.js.org) for particle animations will be accepted**. If it's not using tsParticles, the PR will be closed and marked as invalid. If more than one is invalid, all the invalid PR will be marked as **spam**.

Requirements for posting a template:

- Add all files in a folder, the template **MUST** work.
- Add a screenshot in the *__screenshots* (https://github.com/tsparticles/404-templates/tree/main/__screenshots) folder
- Add the template to `data/pages.json` (https://github.com/tsparticles/404-templates/blob/main/data/pages.json) file. You can copy an existing one updating all the informations.

All the *Pull Requests* require this format. Any wrong Pull Request will be marked as invalid and closed.

**If I receive multiple invalid Pull Requests from the same user, I'll mark them as spam.**

### PR DESCRIPTION
**Pull request by Ahsan-Ehtesham**
Fixes #18 

# Work Done:

- Added responsive 404 desert template

# Relevant Screenshots/Gifs

image

### CODE REVIEW

**Comment by Ahsan-Ehtesham** [comment]
@matteobruni Kindly add hacktoberfest-accepted label after review

**Comment by Ahsan-Ehtesham** [comment]
@matteobruni kindly review

**Review by matteobruni — CHANGES_REQUESTED** [review]
This configuration is a copy-paste of the `ocean` template. As I said in other reviews, I'd like to have different particle configurations, not different CSS

**Comment by Ahsan-Ehtesham** [comment]
> This configuration is a copy-paste of the `ocean` template. As I said in other reviews, I'd like to have different particle configurations, not different CSS

Okay sure, I will change particle configurations

**Comment by Ahsan-Ehtesham** [comment]
Updated particles configuration plus solved all merge conflicts, ready to merge if accepted

**Review by matteobruni — CHANGES_REQUESTED** [review]
The index is broken now, please test everything before asking for a review.

**Comment by Ahsan-Ehtesham** [comment]
> The index is broken now, please test everything before asking for a review.

Sorry for that, I have updated it, you can check the screenshots
image
image
The code I fixed in index.html:
image

File: data/pages.json [hunk 1]
```diff
@@ -103,5 +103,10 @@
         "description": "Imposter 404",
         "imageUrl": "https://raw.githubusercontent.com/Rohanfizz/404-templates/main/__screenshots/imposter.png?raw=true",
         "url": "imposter/404.html"
-    }
+    },
+    {
+        "description": "Desert 404",
+        "imageUrl": "/__screenshots/desert.png?raw=true",
```
Comment (matteobruni) [inline data/pages.json:109]: The imageUrl format must be like all the other ones

**Review by matteobruni — CHANGES_REQUESTED** [review]

File: data/pages.json (hunk 1 above)
Comment (matteobruni) [reply data/pages.json]: It's correct that the image is not visible locally, it's not in the repository yet. It will work when pushed.

Comment (matteobruni) [reply data/pages.json]: The URL is just a string, I don't think that changing the value is breaking the index.html, maybe there's another error.

Comment (Ahsan-Ehtesham) [reply data/pages.json]: I resolved the merge conflict as well, so now it shall work

**Review by matteobruni — CHANGES_REQUESTED** [review]
The main index is still not working. Please test the whole website.

**Comment by Ahsan-Ehtesham** [comment]
> The main index is still not working. Please test the whole website.

Can you share the screenshots how it is not working, as I have done everything possible at my side

**Comment by Ahsan-Ehtesham** [comment]
Creating a new PR

**Comment by matteobruni** [comment]
Screenshot 2022-10-27 alle 19 51 35

**Comment by Ahsan-Ehtesham** [comment]
> Screenshot 2022-10-27 alle 19 51 35

#84 is my new updated PR

**Pull request closed without merging**